crates/sys/src/notification.rs

```diff
@@ -0,0 +1,95 @@
+//! Low-level bindings to Notificcation API.
+
+use core::ffi::c_char;
+use crate::{opaque, c_string};
+
+pub const RECORD_NOTIFICATION: *const c_char = c_string!("notification");
+
+opaque!(Notification);
+opaque!(NotificationSequence);
+
+extern "C" {
+    #[link_name="notification_message"]
+    pub fn message(notifications_app: *mut Notification, sequence: *const NotificationSequence);
+}
+
+pub mod vibro {
+    use super::NotificationSequence;
+
+    extern {
+        #[link_name="sequence_set_vibro_on"]
+        pub static ON: NotificationSequence;
+        #[link_name="sequence_reset_vibro"]
+        pub static OFF: NotificationSequence;
+    }
+}
+
+pub mod led {
+    use super::NotificationSequence;
+
+    pub mod red {
+        use super::NotificationSequence;
+
+        extern {
+            #[link_name="sequence_blink_red_10"]
+            pub static BLINK: NotificationSequence;
+            #[link_name="sequence_blink_red_100"]
+            pub static FLASH: NotificationSequence;
```

Comment:
I'm wondering if we should go with `BLINK_10` and `BLINK_100` to match the internal symbols (at least in the `-sys` crate).

Reply:
Yeah that makes sense. Done.